transcom/mymove
# B-20235

## [Agility ticket](https://github.com/transcom/mymove/pull/13090)

## Summary
It was requested that until the work is complete to send all completed ppms to "CLOSEOUT_COMPLETE" status, we display the feedback if any items were rejected, regardless of document review status.

How to test
1. Get a ppm to document review stage
2. Login as an SC and reject a document
3. Login as customer and confirm you can see the View Feedback button
4. Try and break
